partiql-parser/src/main/antlr/PartiQL.g4

```diff
@@ -100,17 +99,43 @@ tableDef
 
 tableDefPart
     : columnName type columnConstraint*                             # ColumnDeclaration
+    | ( CONSTRAINT constraintName )?  tableConstraintDef            # TableConstrDeclaration
+    ;
+
+tableConstraintDef
+    : checkConstraintDef                                            # TableConstrCheck
+    | uniqueConstraintDef                                           # TableConstrUnique
     ;
 
 columnConstraint
-    : ( CONSTRAINT columnConstraintName )?  columnConstraintDef
+    : ( CONSTRAINT constraintName )?  columnConstraintDef
     ;
 
 columnConstraintDef
-    : NOT NULL                                  # ColConstrNotNull
-    | NULL                                      # ColConstrNull
+    : NOT NULL                                     # ColConstrNotNull
```

Comment:
Since the partiql-ast now uses "attribute" in the table definition, should the ANTLR definitions also use "attribute" rather than "column"? I think we should keep the parser and ast terminology consistent.

Related -- would the PIG parser also need to be updated to use "attribute"?

Reply:
I think this is a valid call out. But given the fact that Antlr grammar is an internal concern, perhaps we can treat this as a backlog item? 

I don't think we want to introduce changes to the PIG parser.

Reply:
At some point we would want to update the ANTLR grammar on our [website](https://partiql.org/syntax/antlr.html). Perhaps then would be a good time to do those changes in batch?

Reply:
> I think this is a valid call out. But given the fact that Antlr grammar is an internal concern, perhaps we can treat this as a backlog item?

Ah, I hadn't realized the current ANTLR-generated parse code was meant to be "internal" despite the visibility modifiers indicating it was "public". Created an issue to track the internalization of the ANTLR-generated sources -- https://github.com/partiql/partiql-lang-kotlin/issues/1446.

> At some point we would want to update the ANTLR grammar on our [website](https://partiql.org/syntax/antlr.html). Perhaps then would be a good time to do those changes in batch?

Since we'll likely have other changes to the ANTLR grammar before the DDL syntax is finalized, I'm fine w/ punting on the changes. Could you create an issue to track updating the website grammar and grammar within `partiql-parser`? That should make it easier to track what'll need changed before the next release.